src/modules/commerce/index.ts

```diff
@@ -266,4 +338,84 @@ export class CommerceModule {
       this.faker.definitions.commerce.product_description
     );
   }
+
+  /**
+   * Returns a random [ISBN](https://en.wikipedia.org/wiki/ISBN) identifier.
+   *
+   * @param options The variant to return or an options object. Defaults to `{}`.
+   * @param options.variant The variant to return. Can be either `10` (10-digit format)
+   * or `13` (13-digit format). Defaults to `13`.
+   * @param options.separator The separator to use in the format. Defaults to `'-'`.
+   *
+   * @example
+   * faker.commerce.isbn() // '978-0-692-82459-7'
+   * faker.commerce.isbn(10) // '1-155-36404-X'
+   * faker.commerce.isbn(13) // '978-1-60808-867-6'
+   * faker.commerce.isbn({ separator: ' ' }) // '978 0 452 81498 1'
+   * faker.commerce.isbn({ variant: 10, separator: ' ' }) // '0 940319 49 7'
+   * faker.commerce.isbn({ variant: 13, separator: ' ' }) // '978 1 6618 9122 0'
+   *
+   * @since 8.0.0
+   */
+  isbn(
+    options:
+      | 10
+      | 13
+      | {
+          /**
+           * The variant of the identifier to return.
+           * Can be either `10` (10-digit format)
+           * or `13` (13-digit format).
+           *
+           * @default 13
+           */
+          variant?: 10 | 13;
+
+          /**
+           * The separator to use in the format.
+           *
+           * @default '-'
+           */
+          separator?: string;
+        } = {}
+  ): string {
+    if (typeof options === 'number') {
+      options = { variant: options };
+    }
+
+    const { variant = 13, separator = '-' } = options;
+
+    const prefix = '978';
+    const group: string = this.faker.helpers
+      .objectKey(ISBN_LENGTH_RULES)
+      .toString();
+    const element: string = this.faker.string.numeric(8);
```

Comment:
IMO there is no need to explicitly define that this is a string here.

```suggestion
    const element = this.faker.string.numeric(8);
```